xbmc/video/dialogs/GUIDialogSubtitleSettings.cpp

```diff
@@ -116,9 +116,13 @@ std::string CGUIDialogSubtitleSettings::BrowseForSubtitle()
 
   std::string strPath;
   if (URIUtils::IsInRAR(g_application.CurrentFileItem().GetPath()) || URIUtils::IsInZIP(g_application.CurrentFileItem().GetPath()))
+  {
     strPath = CURL(g_application.CurrentFileItem().GetPath()).GetHostName();
-  else
+  }
+  else if (!URIUtils::IsPlugin(g_application.CurrentFileItem().GetPath()))
+  {
     strPath = g_application.CurrentFileItem().GetPath();
+  }
```

Comment:
Makes sense :+1: 

Maybe it would be good then to prevent `paths.push_back(URIUtils::GetDirectory(strPath));` below with a `if (!strPath.empty())` if not set before?

Reply:
Do you mean `shares.push_back(share);` if the share path is a `plugin://` path? I think that's a valid use-case. Media paths have to be inserted by the user, so if the user wants to add a media share that runs a plugin path I don't think we should prevent it.

Reply:
Ah, I see what you mean now, you're right.